deepface/basemodels/Facenet.py

```diff
@@ -550,4 +550,8 @@ def loadModel():
 	
 	#-----------------------------------
 	
-	return model
\ No newline at end of file
+	return model
+
+
+def name():
```

Comment:
In plaats van aan al deze modules een `name()` toe te voegen, kun je ook gebruik maken van de `BaseModel` enum die nu in `finetuning.py` staat (die zou eigenlijk ergens anders op een wat generiekere plek moeten). Daar is voor elk model een enum item aangemaakt waarvan je ook de naam op kunt vragen. En elke `BaseModel` heeft ook een methode `load_embedding_model()` die je kunt gebruiken, zo hebben we 1 API waarmee we onze modellen kunnen beheren en hoeven we niet afhankelijk te zijn van of er wel of niet een functie is gedefinieerd in 1 van de deepface bestanden.

Reply:
ok, deepface_lib_model gemerged met base model, nu in de models.py module. overigens ruilen we nu wel verwachtingen over 'name()' in voor andere verwachtingen zoals loadModel(). heb voor de duidelijkheid  load_embedding_model laten staan als wrapper om property model heen, maar zouden die ook nog kunnen mergen